MedicalDocManagment.WebUI/app/main/main-app.component.ts

```diff
@@ -3,7 +3,7 @@
 @Component({
     moduleId: module.id,
     selector: 'main-app-component',
-    templateUrl: 'main-app.component.html'
+    templateUrl: 'main-app.component.html',
 })
```

Comment:
Is this works with comma at the end of line `templateUrl: 'main-app.component.html',`?